src/components/ImageSlider/ImageList.js

```diff
@@ -101,7 +101,7 @@ class ImageList extends Component {
       autoplay: { enabled: enableAutoplay },
     } = this.props
     const { scrollAction } = images[index]
-    if (scrollAction) scrollAction(index)
+    if (scrollAction === "function") scrollAction(index)
```

Comment:
I think the better way to test this would be to:
```suggestion
    if (scrollAction && Object.prototype.toString.call(scrollAction) === '[object Function]') scrollAction(index)
```

Reply:
Definitely super strange that it's sending the placeholder image url instead though... maybe there's a deeper issue?

Reply:
Why not just add `typeof` at the start to check if it's a function?
```suggestion
    if (typeof scrollAction === "function") scrollAction(index)
```

Reply:
I was reading that `typeof` can return `function` in several different contexts, but if you know it works I trust you

Reply:
I haven't heard of `typeof function` returning in different contexts, but I do know `typeof object` does return from things like `Array`s

Reply:
+1 for `typeof function`

Reply:
@scott-massey it's weird... but it's happening 🤷🏻‍♂️
<img width="681" alt="Captura de Tela 2021-02-04 às 15 11 42" src="https://user-images.githubusercontent.com/22667080/106949770-06141e80-66f3-11eb-8c13-f54013386a8d.png">